features/storage/nvstore/TESTS/nvstore/functionality/main.cpp

```diff
@@ -27,10 +31,6 @@
 #include <stdio.h>
 #include <algorithm>
 
-#if !NVSTORE_ENABLED
-#error [NOT_SUPPORTED] NVSTORE needs to be enabled for this test
-#else
```

Comment:
Again, I don't see the point of moving the `#if` to before `#include` as `nvstore.h` might disable that flag.

Reply:
This macro comes from "features/storage/nvstore/mbed_lib.json".i think this is fine to keep at the beginning

Reply:
Not totally true. It can be disabled, if FLASH is not enabled:
https://github.com/ARMmbed/mbed-os/blob/master/features/storage/nvstore/source/nvstore.h#L23

Reply:
Yes, I noticed, I will move this after nvstore.h file